src/modules/27-platform/secrets/components/CreateUpdateSecret/__tests__/CreateUpdateSecret.test.tsx

```diff
@@ -94,6 +124,9 @@ describe('CreateUpdateSecret', () => {
     await waitFor(() => expect(getByText('platform.secrets.secret.labelSecretType')).toBeTruthy())
     expect(getByText('platform.secrets.secret.labelSecretType')).toBeDefined()
     expect(getByText('platform.secrets.labelValue')).toBeDefined()
+    const secretManagerField = screen.getByTestId(/secretManagerIdentifier/)
+    expect(secretManagerField).toBeInTheDocument()
+    expect(secretManagerField).not.toBeDisabled()
```

Comment:
Nothing wrong here, but since `getBy` throws when the target doesn't exist, you can simplify to a single line and get the same result.
```suggestion
    expect(screen.getByTestId(/secretManagerIdentifier/)).not.toBeDisabled()
```